app/assets/src/api/index.js

```diff
@@ -341,7 +341,7 @@ const getSamplesLocations = ({ domain, filters, projectId, search }) =>
 const getSamplePipelineResults = id =>
   get(`/samples/${id}/results_folder.json`);
 
-const getBulkDownloadTypes = () => get("/bulk_downloads/types");
+const getBulkDownloadTypeMetadata = () => get("/bulk_downloads/type_metadata");
```

Comment:
Renamed this "type" to "type metadata" to distinguish it from the "type" field of a bulk download object.
"type metadata" is an object with fields like "description", "display name" for that particular download type.
"type" on a bulk download object is a string such as "sample_taxons_summary" which specifies the download type.

Reply:
But what does this return? Isn't it all the types + their metadata?